src/pcapi/alembic/versions/20210329_202ae94f2c8f_add_internal_id_to_allocine_venue_.py

```diff
@@ -0,0 +1,26 @@
+"""add_internal_id_to_allocine_venue_provider
+
+Revision ID: 202ae94f2c8f
+Revises: 5a767131f3ec
+Create Date: 2021-03-29 09:27:06.343973
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "202ae94f2c8f"
+down_revision = "5a767131f3ec"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.add_column("allocine_venue_provider", sa.Column("internalId", sa.String(length=20), nullable=True))
```

Comment:
```suggestion
    op.add_column("allocine_venue_provider", sa.Column("internalId", sa.Text(), nullable=True))
```

L'intérêt de limiter la longueur du champ me semble assez limité. Niveau performance (dans notre contexte), une colonne `TEXT` ne change rien. Et, surtout, on n'aura pas besoin de faire une migration si un jour 20 caractères ne sont plus suffisants. :)